pdr_backend/models/test/test_base_config.py

```diff
@@ -4,7 +4,7 @@
 
 from pdr_backend.models.base_config import BaseConfig
 
-PRIV_KEY = "0xef4b441145c1d0f3b4bc6d61d29f5c6e502359481152f869247c7a4244d45209"
+PRIV_KEY = "0xc594c6e5def4bab63ac29eed19a134c130388f74f019bc74b8f4389df2837a58"
```

Comment:
for everything inside tests, you can use the key defined in pytest.ini as an env var

Reply:
Good point. Done